index.html

```diff
@@ -185,42 +184,50 @@ <h3>Skills</h3>
                     </ul>
                 </div>
             </section>
-        </div>
 
-        <div class="wrapper-menu" id="wrapper-menu">
-            <header class="header">
-                <nav class="nav">
-                    <img src="img/image_geometry_menu_1.svg" alt="">
-                    <a href="#" onclick="closeMenu()"><img class="menu-button" src="img/Union.svg" alt="Menu Buttom"></a>
-                </nav>
-            </header>
+            <section class="contact">
+                <div class="contact-form">
+                    <div class="title set">
+                        <h2>Get started</h2>
+                    </div>
+                    <div class="content set">
+                        <p class="description">
+                            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna.
+                        </p>
+                        <form class="user-form" action="https://formspree.io/f/xoqrloon" method="post">
+                            <label for="name">
+                                Your name
+                                <input class="name-font" type="text" name="name" id="name" maxlength="30" placeholder="Enter your name" required>
+                            </label>
+                            <label for="email" id="email_label">
+                                Your email
+                                <input type="email" name="email" id="email" placeholder="Enter your email address" required>
+                            </label>
+                            <textarea id="message" name="message" placeholder="Enter your message here..." maxlength="500" required></textarea>
 
-            <section class="menu">
-                <div class="content-menu">
-                    <ul class="menu-list">
-                        <li>Hello</li>
-                        <li>Portfolio</li>
-                        <li>About</li>
-                        <li>Contact</li>
-                    </ul>
-                </div>
-                <div class="content-bottom">
-                    <img class="f-img" src="img/image_geometry_menu_2.svg" alt="">
+                            <button class="start-button" type="submit">
+                                Start collaboration
+                            </button>
```

Comment:
- [ ] I think you forgot to check on this button interaction as suggested by the previous reviewer. I noticed you used the wrong hovering colour. It shouldn't be red. 
Please take a look at the original design:

Original Design |
:--------------------:|
![](https://user-images.githubusercontent.com/43995830/159681502-d094e823-5121-48ef-aa75-43aa3c2c8688.png)

Test Demo of your app|
:-----------------------------:|
![a1](https://user-images.githubusercontent.com/39523312/159703079-e278f648-0160-4a1b-8ec1-56e6c7e75d4b.gif)